app/models/token.rb

```diff
@@ -42,7 +42,7 @@ def jwt_string
 
   # Decode the json web token when we receive it from the client
   def token
-    @token ||= JWT.decode(jwt_string, secret_key, hmac_sha_algo)
+    @token ||= JWT.decode(jwt_string, secret_key, true, {verify_expiration: false})
```

Comment:
Line is too long. [83/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.